### ISSUE TITLE
Operator keywords in column names are trimmed off from orginal colum names

### ISSUE THREAD

**Issue by lpatoyst** [dmfay/massive-js#426]
Hi, 

We are implementing massive v2.7.0 since yesterday. We have an issue with the following query : 

```
DB.notifications.find({
            "transaction_id": objectA,
            "content->>'pspReference'": objectB,
            "content->>'eventCode'": 'string',
            is_sent: true
          }, {
            columns: ['id'],
            limit: 1
          }, (err, notifications) => {...})
```

Generates the following SQL query:

```
SELECT id FROM "notifications" 
WHERE "transaction_id" = $1 
AND "content"->>'pspReference' = $2 
AND "content"->>'eventCode' = $3 
AND "is" IS $4 order by "id" limit 1
```

showing that column `is_sent` has been truncated to `is`.

We had a look at the source code and highly suggest : 
- to tell us if there is anything wrong with our payload
- else to have a look at https://github.com/dmfay/massive-js/blob/v2/lib/where.js and https://github.com/dmfay/massive-js/blob/v2/lib/operations_map.js
- to consider having regex filtering whole words, not only find the index of the keywords (as columns with `between`, `like` or `is` in their names may exists, as in our case)
- to consider cases of operator keywords having more than one spaces (as in `not    in`), which may also fail to be recognized (to be confirmed).
- to make tests on columns names containing those same keywords.

Meanwhile we are reverting and fixing our version of massive to 2.6.1.

Regards,

**Comment by dmfay** [comment]
Your payload is fine; that's a bug. Easy enough to test for; I'll take your ideas for fixing it under consideration :)

**Comment by dmfay** [comment]
I've just published v2.7.1 with a revision of the criteria key parsing logic. Give it a whirl and let me know if you run into any problems. I'll be porting it to master shortly (thinking about simplifying JSON traversal to `column.path.to.json.field` instead of making users deal with `->>` and `#>>`), since a similar logic change happened in v3 as well but hasn't been publicly released yet.

**Comment by lpatoyst** [comment]
Using 2.7.1 as of now. Good work :+1: Thanks @dmfay :)
